addons/l10n_es/data/template/account.tax-es_common.csv

```diff
@@ -1,523 +1,523 @@
-"","","","","","","","","","","","","tax","l10n_es.mod_303_15","refund","account_common_477","-100",""
-"account_tax_template_p_iva10_isp_bi","","VAT 10% reverse charge (investment goods)","","purchase","10% IG RC","IVA 10% ISP (bienes de inversión)","10.0","percent","tax_group_iva_10","","","base","l10n_es.mod_303_30,l10n_es.mod_303_12","invoice","","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_31","invoice","account_common_472","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_13","invoice","account_common_477","-100",""
-"","","","","","","","","","","","","base","l10n_es.mod_303_40,l10n_es.mod_303_14_purchase","refund","","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_41","refund","account_common_472","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_15","refund","account_common_477","-100",""
-"account_tax_template_p_iva21_isp_bi","","VAT 21% reverse charge (investment goods)","","purchase","21% IG RC","IVA 21% ISP (bienes de inversión)","21.0","percent","tax_group_iva_21","","","base","l10n_es.mod_303_30,l10n_es.mod_303_12","invoice","","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_31","invoice","account_common_472","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_13","invoice","account_common_477","-100",""
-"","","","","","","","","","","","","base","l10n_es.mod_303_40,l10n_es.mod_303_14_purchase","refund","","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_41","refund","account_common_472","100",""
-"","","","","","","","","","","","","tax","l10n_es.mod_303_15","refund","account_common_477","-100",""
-"account_tax_template_p_rp19","","19% Withholding (loans)","","purchase","19% WH L","Retenciones 19% (préstamos)","-19.0","percent","tax_group_retenciones_19","","","base","","invoice","","",""
-"","","","","","","","","","","","","tax","","invoice","account_common_4751","",""
-"","","","","","","","","","","","","base","","refund","","",""
-"","","","","","","","","","","","","tax","","refund","account_common_4751","",""
-"account_tax_template_p_rrD19","","19% Withholding (dividend distribution)","","purchase","19% WH DD","Retenciones 19% (reparto de dividendos)","-19.0","percent","tax_group_retenciones_19","","","base","","invoice","","",""
-"","","","","","","","","","","","","tax","","invoice","account_common_4751","",""
-"","","","","","","","","","","","","base","","refund","","",""
-"","","","","","","","","","","","","tax","","refund","account_common_4751","",""
+"id","sequence","description","invoice_label","type_tax_use","name","description@es","amount","amount_type","tax_group_id","analytic","include_base_amount","repartition_line_ids/repartition_type","repartition_line_ids/tag_ids","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent"

```

Comment:
@casp-odoo Should not this invoice_label@es be added with the rebase?  (just not to forget it)

Reply:
@jco-odoo I don't really understand, could you please give me a little more context ? 
@moerradi did a lot of the changes on this file, he might also be in this situation

Reply:
in my PR  `invoice_label@es` is still present

Reply:
@jco-odoo up? :)

Reply:
@casp-odoo Just rebase and compare your branch with master to see if you do not delete anything by accident.  (it was the only point I wanted to make)